odk1-src/central-forms.rst

```diff
@@ -122,6 +161,40 @@ To set the form lifecycle stage, go to the :ref:`Form Access <central-projects-f
 
 You can find more information about the Form Access page :ref:`here <central-projects-form-access>`.
 
+.. _central-forms-updates:
+
+Form Updates
+------------
+
+As of Central 0.8, it is possible to update a published Form with a new design definition, or new Media Files, and to test these changes before they are applied to the Form in use.
+
+There is one primary restriction Central enforces on updated design definitions: once defined in a published Form version, each field Data Name (in technical terms, the Instance XPath) cannot change its Data Type. Unused fields may be removed, and new fields may be added, but if any field reuses a previously existing Data Name, it must have the same Type as it did before. If you run into an error with this restriction, the easiest solution is usually to rename the changed field to a new name.
+
+To begin the process of updating a published Form, click on the :guilabel:`Create a new Draft` button in the Draft navigation on the Form:
+
+   .. image:: /img/central-forms/update-form.png
+
+Initially, the new Draft will have the same design definition as the published Form. If you only want to update attachment Media Files, this means you don't have to bother uploading a definition at all: you can go straight to the :guilabel:`Media Files` tab and :ref:`upload the changed files <central-forms-attachments>`.
+
+You can replace the Draft definition, Media Files, and make test submissions as with the :ref:`initial Form Draft <central-forms-draft>` before the Form was first published. Test submissions will not interfere with published Form submissions.
+
+Once you are satisfied that your updated Form is ready to be published for immediate use, you can click on the :guilabel:`Publish Draft` button on the Draft Status tab.
+
+.. admonition:: Form Version naming
+
+  If you did not change the design definition, or your updated design definition did not change the ``version`` of the Form, Central will not be able to publish the Form as-is. This is because the ``version`` must change in order for data collection clients to understand that they should update. You can upload a new design definition with a changed ``version``, or else Central will offer to change it in-place for you.
+
+.. _central-forms-versions:
+
+Older Form Versions
+-------------------
+
+If you have published multiple version of a Form, you can each of them under the :guilabel:`Versions` tab.
+
+   .. image:: /img/central-forms/versions.png
+
+Each published version of the Form will be listed, along with actions to download the design definition of each Form. In future versions of Central, the Media File attachments associated with each version of the Form will be downloadable as well.
```

Comment:
"design definition" -> "form definition?"

Reply:
i wasn't sure "form definition" was a recognizable term

Reply:
Ah, so you were actually intending the phrase "design definition of each Form?" I think the word "design" there really feels out of place. The "definition of each Form" would be ok by me as well.